database/migrations/2022_12_01_145035_create_employee_has_classes_table.php

```diff
@@ -13,11 +13,11 @@
      */
     public function up()
     {
-        Schema::create('teacher_has_classes', function (Blueprint $table) {
+        Schema::create('employee_has_classes', function (Blueprint $table) {
```

Comment:
Vergeet de naam van het bestand ook niet te veranderen naar `employees` want dat is nu nog `create_teachers_has_classes_table`